File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.java
class Solution {
    public String solution(String[] id_pw, String[][] db) {
        String answer = "";
        
        for(int i = 0; i < db.length; i++){
            String id = id_pw[0];
            String pw = id_pw[1];
            String dbid = db[i][0];
            String dbpw = db[i][1];
            
            if(id.equals(dbid) && pw.equals(dbpw)){
                answer = "login";
                break;
            }else if(id.equals(dbid)){
                answer = "wrong pw";
            }else if(!answer.equals("wrong pw")){
                answer = "fail";
            }
            
            
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    answer = 'fail'
    
    for userId, userPw in db:
        if userId == id_pw[0]:
            if userPw == id_pw[1]:
                answer = 'login'
                break
            else:
                answer = 'wrong pw'
                break
        
    return answer